### ISSUE TITLE
Add GregTech

### ISSUE THREAD

**Issue by fusionmc-evilscoop** [TagnumElite/ProjectE-Integration#106]
Can you please add support for gregtech modern - 
https://github.com/GregTechCEu/GregTech-Modern

**Comment by ProgrammerFailure** [comment]
I would also like this feature, would be helpful for my nomifactory playthrough.

**Comment by albatrossrus** [comment]
I totally agree, there's not enough GTCEU support out there, especially when it comes to using tools. It doesn't creates an EMC on item crafted.

**Comment by argon1321** [comment]
Please do so!
